### PR TITLE
W-16102873 RPA Manager release notes 2024-06/2.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
+// == What's New
+//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| RPA Builder is now able to download runtime packages from RPA Manager. | W-16080683
```
Comment (sebermuley) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Updating the WebDriver while testing the Chrome Web Session or Edge Web Session action steps in RPA Builder no longer fails. RPA Builder is now able to download runtime packages from RPA Manager. | W-16080683
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,32 @@
...
+| Serialized JSON in an activity parameter of type string will not been modified during the execution of a process. | W-15881024
```
Comment (sebermuley) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The first character of an Activity Parameter string values in JSON format no longer changes to lower case when passed from one Activity to another.
 | W-15881024
```

Comment (sebermuley) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: Is this correct? Does this only happen for JSON format or for all strings or for all characters following a curly bracket?

Comment (jhammann72) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: If we want to describe this in that detail I need to check it with the engineers.

Comment (sebermuley) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: We only need as much detail as absolutely necessary for mentioning the behavior the users experienced (not the cause of it).
We could probably shorten it like this:
Activity Parameter values in JSON format no longer change when passed from one Activity to another.

**Review by sebermuley — APPROVED** [review]

**Pull request merged**